resources/js/pages/ErrorPage.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="d-flex">
+        <div class="w-50-flex center">
+            <h2>Page not found</h2>
```

Comment:
you could put this into the en.json now, since we'd have to put it there later anyway